src/controllers/auth.js

```diff
@@ -71,47 +88,55 @@ async function confirmed (req, res, next, client = FxAOAuthClient) {
   // Check if user is signing up or signing in,
   // then add new users to db and send email.
   if (!existingUser) {
-    // req.session.newUser determines whether or not we show "fxa_new_user_bar" in template
+    // req.session.newUser determines whether or not we show `fxa_new_user_bar`
+    // in template
     req.session.newUser = true
     const signupLanguage = req.locale
-    const verifiedSubscriber = await addSubscriber(email, signupLanguage, fxaUser.accessToken, fxaUser.refreshToken, fxaProfileData)
-
-    // TODO:
-    // duping some of user/verify for now
-    // let unsafeBreachesForEmail = []
-
-    // unsafeBreachesForEmail = await getBreachesForEmail(
-    //   sha1(email),
-    //   req.app.locals.breaches,
-    //   true
-    // )
-
-    // const utmID = 'report'
-    // const reportSubject = unsafeBreachesForEmail.length ? req.fluentFormat('email-subject-found-breaches') : req.fluentFormat('email-subject-no-breaches')
-
-    // await sendEmail(
-    //   email,
-    //   reportSubject,
-    //   'email-2022',
-    //   {
-    //     supportedLocales: req.supportedLocales,
-    //     breachedEmail: email,
-    //     recipientEmail: email,
-    //     date: req.fluentFormat(new Date()),
-    //     unsafeBreachesForEmail,
-    //     ctaHref: getEmailCtaHref(utmID, 'dashboard-cta'),
-    //     utmCampaign: utmID,
-    //     unsubscribeUrl: getUnsubscribeUrl(verifiedSubscriber, utmID),
-    //     whichPartial: 'email_partials/report',
-    //     heading: req.fluentFormat('email-breach-summary')
-    //   }
-    // )
+    const verifiedSubscriber = await addSubscriber(
+      email,
+      signupLanguage,
+      fxaUser.accessToken,
+      fxaUser.refreshToken,
+      fxaProfileData
+    )
+
+    // Get breaches for email the user signed-up with
+    const allBreaches = req.app.locals.breaches
+    const unsafeBreachesForEmail = await getBreachesForEmail(
+      email.sha1,
+      allBreaches,
+      true,
+      false
```

Comment:
is there a reason why we'd want to not filter?

Reply:
No, I think we should filter. I rushed going over the args and think I mistakenly thought it was for sorting and not filtering — fixed!